src/main/java/com/mineinabyss/looty/ecs/systems/CooldownDisplaySystem.kt

```diff
@@ -43,8 +44,10 @@ object CooldownDisplaySystem : TickingSystem(interval = INTERVAL) {
                     repeat(squaresLeft) {
                         append(displayChar)
                     }
-                    append(ChatColor.GRAY)
-                    append(" [$timeLeft]")
+                    if (timeLeft.toDouble(DurationUnit.MILLISECONDS) < 250) append(
```

Comment:
This should only fire exactly when the cooldown is done. I think just scheduling a task is the best option here (i.e. check if it's the last iteration for this cooldown item and from there schedule a task that will run in timeLeft ticks.)